packages/create-app/template-preact-ts/package.json

```diff
@@ -10,8 +10,9 @@
     "preact": "^10.5.9"
   },
   "devDependencies": {
+    "@preact/preset-vite": "^1.0.0",
```

Comment:
Shouldn't this be 2.0.0?

And `@prefresh/vite` isn't required here, as its implicitly imported by @preact/preset-vite

Reply:
Good point, applied those changes 👍